packages/editor-ui/src/modules/nodeTypes.ts

```diff
@@ -27,23 +27,26 @@ const module: Module<INodeTypesState, IRootState> = {
 	getters: {
 		allNodeTypes: (state): INodeTypeDescription[] => {
 			return Object.values(state.nodeTypes).reduce<INodeTypeDescription[]>((allNodeTypes, nodeType) => {
-				const nodeVersions = Object.keys(nodeType).map(Number);
+				const versionNumbers = Object.keys(nodeType).map(Number);
+				const allNodeVersions = versionNumbers.map(version => nodeType[version]);
 
-				return [...allNodeTypes, ...nodeVersions.map(version => nodeType[version])];
+				return [...allNodeTypes, ...allNodeVersions];
+			}, []);
+		},
+		allLatestNodeTypes: (state): INodeTypeDescription[] => {
+			return Object.values(state.nodeTypes).reduce<INodeTypeDescription[]>((allLatestNodeTypes, nodeVersions) => {
+				const versionNumbers = Object.keys(nodeVersions).map(Number);
+				const latestNodeVersion = nodeVersions[Math.max(...versionNumbers)];
+
+				return [...allLatestNodeTypes, latestNodeVersion];
 			}, []);
 		},
 		getNodeType: (state) => (nodeTypeName: string, version?: number): INodeTypeDescription | null => {
 			const nodeVersions = state.nodeTypes[nodeTypeName];
```

Comment:
could not nodeVersion be undefined?

Reply:
The only way I can think of this can happen is if a node has no `version` key (but according to the typing this is not possible?), meaning `{}` →  `[]` →  `0` →  `undefined`. Add a check just in case.

Reply:
what if node type does not exist? like an unknown custom node or delete community package..

Reply:
Thank you, added check.